syntax = "proto3";

import "effect.proto";

message ChanCheckReq {
  bool transmit = 1;
  uint32 priority = 2;
  uint32 universe = 3;
  uint32 address = 4;
  uint32 level = 5;
  bool per_address_priority = 6;
  EffectSettings effect = 7;
}

message ChanCheckRes {
  bool transmitting = 1;
  uint32 priority = 2;
  uint32 universe = 3;
  uint32 address = 4;
  uint32 level = 5;
  bool per_address_priority = 6;
  EffectSettings effect = 7;
}
